modules/pricer/src/test/java/com/opengamma/strata/pricer/rate/swaption/BlackSwaptionPhysicalProductPricerTest.java

```diff
@@ -176,14 +210,12 @@ public void present_value_payer_receiver_parity() {
     assertEquals(pvLongPay.getAmount() + pvShortRec.getAmount(), pvSwapPay.getAmount(USD).getAmount(), TOLERANCE_PV);
   }
   
-  @Test
   public void present_value_after_expiry() {
     CurrencyAmount pv = PRICER_SWAPTION_BLACK.presentValue(SWAPTION_PAST, MULTI_USD, BLACK_VOL_CST_SWAPTION_PROVIDER_USD);
     assertEquals(pv.getAmount(), 0.0d, TOLERANCE_PV);
   }
 
```

Comment:
Add present value on expiry date. This is to make sure that a time to expiry of 0 does not cause problem.